types/k6/options.d.ts

```diff
@@ -439,3 +442,8 @@ export type Scenario =
     | ConstantArrivalRateScenario
     | RampingArrivalRateScenario
     | ExternallyControlledScenario;
+
+export interface ScenarioOptions {
+    /** Browser specific options */
+    browser?: any;
```

Comment:
Will the browser team add definitions here? Otherwise it's not very useful.

Reply:
At this point this is the definition that I am aware of. 

Also AFAIK the whole k6/experimental/browser needs rework after the currently ongoing rewrite.